packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -39,10 +39,14 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
   const config = useConfig() as RegistrationConfig;
   const [target, setTarget] = useState<undefined | string>();
   const { patientUuid: uuidOfPatientToEdit } = useParams();
+  const sourcePatientId = new URLSearchParams(search).get('sourceRecord');
```

Comment:
I don't think this kind of data should be passed through the URL. It's a SPA, we can pass it through some kind of state variable (including the AppContext or something if necessary).